### ISSUE TITLE
HTTP/2 support validation

### ISSUE THREAD

**Issue by shanidgafur** [YellowLabTools/YellowLabTools#193]
I'm noticing an issue where some of my sites which support HTTP/2 do not show up as supporting HTTP/2 through yellowlab (check out http://yellowlab.tools/result/eiew5kjm7t). If I check the site using developer tools on either chrome or firefox, the http/2 support does seem to work.

Is this a known issue?

**Comment by gmetais** [comment]
Hi @shanidgafur,

Yes, this is a known issue : https://github.com/stefanjudis/is-http2/issues/10

What server do you use? Is it an Apache 2.4?

**Comment by shanidgafur** [comment]
It's an Akamai GHost proxy.

**Comment by gmetais** [comment]
We'll have to find an alternative detection solution!

**Comment by shanidgafur** [comment]
@gmetais Looks it's working now.. did you push out a fix?

**Comment by gmetais** [comment]
No, nothing. Maybe they made the change?

**Comment by callebokedal** [comment]
As mentioned by @gmetais, this is a known issue. But how do we come to a change using an APLN-check instead of/as well as NPN?

Example:
```
$ true | openssl s_client -connect telia.se:443 -servername telia.se -alpn 'h2' 2>/dev/null | grep 'ALPN protocol'
ALPN protocol: h2
$
```

**Comment by Besfr** [comment]
Hello, I can confirm the bug still exists.
the YLT test : https://yellowlab.tools/result/f6fzkg0fpn/rule/http2
Keycdn test : 
![screenshot_20181107_092331](https://user-images.githubusercontent.com/152320/48119393-73fce180-e26f-11e8-908e-5fa1b39b56da.png)

Test of the command proposed by @callebokedal 
```
$ true | openssl s_client -connect happyculture.coop:443 -servername happyculture.coop -alpn 'h2' 2>/dev/null | grep 'ALPN protocol'
ALPN protocol: h2
```

**Comment by anarcat** [comment]
same here, with https://anarc.at/

https://vitesse.dev.facil.services/result/f6o9gk117z/rule/http2
https://tools.keycdn.com/http2-test?url=anarc.at

**Comment by davdavf** [comment]
Since this is still an issue, would you perhaps consider updating the infotext on https://yellowlab.tools/result/.../rule/http2 so as to reflect that this particular measurement is unreliable in some cases?

For example, something along the lines of:
"This test is known to misdiagnose some server configurations. You may want to doublecheck this measurement in another tool."

**Comment by gmetais** [comment]
Fixed by #322.
